packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -67,12 +69,12 @@ export const SECONDS_PER_WEEK = 7n * SECONDS_PER_DAY;
  */
 
 /**
- * @typedef {import('@agoric/zoe/src/zoeService/utils.js').StartedInstanceKit<
+ * @typedef {StartedInstanceKit<
```

Comment:
I think only the `@typedef` redefinitions affect generated docs.

I guess this is still cleaner.